lib/tasks/projects.rake

```diff
@@ -194,4 +194,29 @@ namespace :projects do
       end
     end
   end
+
+  desc 'Verify Go Projects'
+  task :verify_go_projects, [:count] => :environment do |_task, args|
+    loop do
+      count ||= 1000
+      start_id = REDIS.get("go:update:latest_updated_id").presence || 0
+      puts "Start id: #{start_id}, count: #{start_id.to_i + count}"
+      projects = Project
+        .where("id > ?", start_id)
+        .order(:id)
+        .limit(count)
+
+      if projects.count.zero?
+        puts "Done!"
+        exit
+      elsif (queue_size = Sidekiq::Queue.new('small').size) > 0
+        puts "Waiting for #{queue_size} jobs..."
+        sleep 5
+      else
+        projects
+          .each { |p| PackageManagerDownloadWorker.perform_async("PackageManager::Go", p.name) }
+        REDIS.set("go:update:latest_updated_id", projects.last.id)
+      end
+    end
```

Comment:
@jsonperl here's what I was thinking: create the loop in the rake task so we can keep it running in tmux, and avoid re-booting the Rails env every invocation.

Reply:
Can't you just do that in bash? I'm not a huge fan of assuming a SIGINT for using this thing or it all gets queued.

`watch -n 60 bundle exec rake projects:verify_go_projects[1000]` would do the trick no?

Reply:
Loop removed